library/registry

```diff
@@ -1,7 +1,30 @@
-Maintainers: David Wu (@davidswu),
-             Ryan Abrams (@caervs)
+Maintainers: Chris Patterson (@chrispat),
+             Bryan Clark (@clarkbw),
+             Cory Snider (@corhere),
+             Hayley Swimelar (@deleteriousEffect),
+             He Weiwei (@heww),
+             João Pereira (@joaodrp),
+             Justin Cormack (@justincormack),
+             Kyle Squizzato (@squizzi),
+             Milos Gajdos (@milosgajdos),
+             Sargun Dhillon (@sargun),
+             Wayne Warren (@waynr),
+             Wang Yan (@wy65701436),
+             Steve Lasker (@stevelasker)
+
```

Comment:
> [...] lines starting with `#` are ignored and "entries" are separated by a blank line.
>
>The first entry is the "global" metadata for the image. 

The failing "Diff Comment" CI is because of this blank line so the single `GitRepo` line below is being considered as a separate entry.